<launch>
    <arg name="duty" default="TeamPlayer"/>
    <rosparam command="load" file="$(find bitbots_body_behaviour)/config/behaviour.yaml"/>
    <rosparam command="load" file="$(find bitbots_body_behaviour)/config/animations.yaml"/>
    <!--<rosparam command="load" file="$(find game_controller)/config/game_controller.yaml"/>-->
    <rosparam param="actionlib_client_sub_queue_size">-1</rosparam>
    <rosparam param="actionlib_server_sub_queue_size">-1</rosparam>
    <node name="body_behaviour" pkg="bitbots_body_behaviour" type="body_behaviour.py" respawn="true"  output="screen" args="$(arg duty)" />
    <include file="$(find bitbots_pathfinding_simple)/launch/pathfinding.launch"/>
</launch>
